src/.ocamlformat

```diff
@@ -0,0 +1,66 @@
+profile=ocamlformat
+quiet=false
+max-iters=10
+comment-check=true
+wrap-fun-args=true
+wrap-comments=false
+type-decl-indent=2
+type-decl=compact
+stritem-extension-indent=0
+space-around-variants=true
+space-around-records=true
+space-around-lists=true
+space-around-arrays=true
+single-case=compact
+sequence-style=separator
+sequence-blank-line=preserve-one
+parse-docstrings=false
+parens-tuple-patterns=multi-line-only
+parens-tuple=always
+parens-ite=false
+ocp-indent-compat=false
+nested-match=wrap
+module-item-spacing=sparse
+max-indent=68
```

Comment:
Does reducing this help any with the indentation of our deeply-nested code? `transaction_pool.ml`, `mina_lib.ml` and `cli_entrypoint.ml` all have spots where this makes it hard to read, would be nice if we could improve those.

Reply:
Changing this to 56 or 48 doesn't seem to make any difference, that I could tell.

The default is 68, so I think I'll leave that.